source/users/custom-metadata.txt

```diff
@@ -712,6 +725,19 @@ Configure Authentication Provider Metadata
          If you successfully configure the provider's metadata
          fields, App Services returns a ``204`` response.
 
+.. note:: Tips for Users with Multiple Linked Authentication Providers  
+
+  - To ensure the most up-to-date metadata, users should re-authenticate after 
+    switching to a different authentication provider. Failure to do so may result 
+    in outdated metadata reflected in the user's table and for requests 
+    using authentication providers.
+  - If a user switches between authentication providers, there may be a 30 minute delay 
+    to update metadata that has the same field name for both authentication 
+    providers. For example, both JWT and API Key authentication providers have 
+    a ``name`` metadata field. There will be a delay to update the ``name`` field 
+    in the user's table if the user authenticates with JWT and then with an API key. 
+    The request is always guaranteed to have the most up-to-date metadata associated 
+    with the authentication provider used.
```

Comment:
I don't think this example is necessary since the case is an edge case and there'd have to be other factors involved for this data to become stale--we don't want to confuse users unnecessarily. Also JWT authentication isn't guaranteed to have a "name" field. 

This also makes it seem like we'll only ever have stale metadata regarding the conflicting field name when in fact, other metadata could possibly be stale (especially if we end up expanding on our caching in the future).
```suggestion
  - If a user switches between authentication providers, there may be a 30 minute delay in propagating metadata to the Users Table and requests using authentication providers. The request is always guaranteed to have the most up-to-date metadata associated with the current authentication provider used.
```